articles/what-is-jenkins/index.md

```diff
@@ -0,0 +1,61 @@
+
+Continuous Integration allows the development teams to detect problems early on. It also automates the process of compiling, testing, and reporting the project source code. When a developer push changes into a repository, the CI server gets notified about the changes. The CI server either pulls the repository after a few minutes to see if there are any changes or the repository actively calls the CI server to let it know when there are changes. It then clones the repository and builds the project. If the project fails it notifies the team. The continuous integration server finally generates various reports such as the line coverage of the project test or style issues found on the [project code](https://www.edureka.co/blog/what-is-jenkins/).
+
+### Why Continuous Integration is needed
+- Using CI developers tend to experience fewer conflicts when integrating code, this is because when code get merged it does not easily break. Also, even if it breaks the problem is easily resolved.
+- A developer may forget to run the project code before checking the changes, on the other hand, the CI server does not forget and the problem is caught early on and resolved.
+- CI supports automation through the process of building or compiling, testing, and reporting to the development team.
+- CI allows continuous delivery to [take place](https://medium.com/the-making-of-whereby/why-continuous-integration-is-so-important-7bb63ba5dc57).
+- Using CI locating and fixing bugs becomes very easy because developers do not have to wait too long to get the test results.
+- Developers are continuously notified about the build status, test results, and how the application is doing.
+
+### Continuous Integration with Jenkins
+Jenkins achieves continuous integration with the help of plugins. It has over a thousand plugins that integrate with every tool in the CI and CD (continuous delivery) toolchain. Jenkins reduces the development cycle since code gets to build and is tested continuously, thus releasing new features to the user faster and with fewer errors. A short time is taken to integrate the code, but before the use of Jenkins, integration was done manually and more time was required. The use of Jenkins enables integration to take place continuously. The use of Jenkins has helped the developers get feedback faster after the code has been integrated, this is helpful because if code breaks they know what the issue is and how it will be resolved.
```

Comment:
You mention thousands of plug-ins in the intro. I would make a bulleted list or numbered list of the top plug-ins and how they add even more value to CI.

How was integration done manually before? Describing how arduous the process is makes the value of CI palpable for the reader.

Run-on sentence at end.